sui/deploy-contract.js

```diff
@@ -404,6 +423,8 @@ const GATEWAY_CMD_OPTIONS = [
     new Option('--previousSigners <previousSigners>', 'number of previous signers to retain').default('15'),
 ];
 
+const ITS_CMD_OPTIONS = [new Option('--itsHubAddress <itsHubAddress>', 'The address of the ITS HUB').env('ITS_HUB_ADDRESS')];
```

Comment:
This should be read from the config `config.axelar.contracts.InterchainTokenService.address` instead of provided by the user, see how `evm/its.js` gets trusted addresses